pages/community/styleguide/aboutguide.md

```diff
@@ -0,0 +1,35 @@
+# AsyncAPI Style Guide
```

Comment:
I think we will also need to add 
`---
title:
description:
weight:
---`
just like we did in tutorials